src/metadata-types.ts

```diff
@@ -2,19 +2,21 @@
 export type AnyType = { new (...args: any[]): any }
 
 export interface TypeMetadata {
-    name: string
-    type: AnyType
-    parameters: Array<TypeMetadata>
+  name: string
+  type: AnyType
+  parameters: Array<TypeMetadata>
+  isNullable?: boolean
+  isEnum?: boolean
```

Comment:
As these fields are boolean, I don't see any reason to make them optional. They can be either false or true. Would you mind making them non-optional please?

Reply:
This was done to reduce the size of the metadata, but that might be a detail.

Reply:
That's a good reason. Thanks!